.github/workflows/ci-pkg-install.yml

```diff
@@ -23,7 +23,46 @@ defaults:
 
 jobs:
 
+  init-pypi:
+    runs-on: ubuntu-20.04
+    steps:
+    - run: mkdir pypi && touch pypi/.placeholder
+    - uses: actions/upload-artifact@v3
+      with:
+        name: ci-packages-${{ github.sha }}
+        path: pypi
+
+  build-pypi:
+    needs: init-pypi
+    # This serves to create packages for potential internal dependencies
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: true
+      max-parallel: 1
+      matrix:
+        pkg: ["lite"]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+      - uses: actions/download-artifact@v3
+        with:
+          name: ci-packages-${{ github.sha }}
+          path: pypi
+      - name: Build package
+        env:
+          PACKAGE_NAME: ${{ matrix.pkg }}
+        run: |
+          python setup.py sdist
+          cp dist/* pypi/
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ci-packages-${{ github.sha }}
+          path: pypi
+
   install-pkg:
+    needs: build-pypi
```

Comment:
Because Pl now needs to have Lite as of dependency, the other way would be building all packages in sequence and aggregating them, this is a way how to speed this up and be able to run all in parallel...